cpu/samd5x/periph/can.c

```diff
@@ -427,19 +463,36 @@ static int _init(candev_t *candev)
     if (IS_ACTIVE(ENABLE_DEBUG)) {
         _dump_msg_ram_section(dev);
     }
-    /* Disable automatic retransmission by default */
-    /* This can be added as a configuration parameter for the CAN controller */
-    dev->conf->can->CCCR.reg |= CAN_CCCR_DAR;
+
+    uint32_t cccr = dev->conf->can->CCCR.reg;
+    cccr &= ~(CAN_CCCR_DAR | CAN_CCCR_TXP | CAN_CCCR_MON);
+
+    if (dev->conf->disable_automatic_retransmission) {
+         cccr |= CAN_CCCR_DAR;
+    }
+
+    if (!dev->conf->disable_transmit_pause) {
```

Comment:
transmit_pause  seems to an non standard extra this can controller provides. it should be enabled if needed.